### PR TITLE
Only set min-height on main if mktNavEnabled

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1151456

screenshot 2015-04-07 10 06 25
screenshot 2015-04-07 10 07 34

### CODE REVIEW

File: src/media/js/elements/nav.js [hunk 1]
```diff
@@ -241,9 +241,11 @@ define('elements/nav',
         // wouldn't apply unless <html>/<body> had heights, and setting to
         // 100% height would mess up the page since our <footer> isn't part of
         // <main>. Could be resolved if we made <footer> a part of <main>.
-        var main = document.querySelector('main');
-        if (main) {
-            document.querySelector('main').style.minHeight = screen.height + 'px';
+        if (settings.mktNavEnabled) {
+            var main = document.querySelector('main');
+            if (main) {
+                main.style.minHeight = screen.height + 'px';
```
Comment (chuckharmston) [inline src/media/js/elements/nav.js]: Probably easier to just jQuery this, it's already imported and you don't need to worry about units or other edge cases.

**Comment by chuckharmston** [comment]
Test failure looks unrelated.

**Comment by chuckharmston** [comment]
r+wc

File: src/media/js/elements/nav.js [hunk 2]
```diff
@@ -241,9 +241,11 @@ define('elements/nav',
         // wouldn't apply unless <html>/<body> had heights, and setting to
         // 100% height would mess up the page since our <footer> isn't part of
         // <main>. Could be resolved if we made <footer> a part of <main>.
-        var main = document.querySelector('main');
-        if (main) {
-            document.querySelector('main').style.minHeight = screen.height + 'px';
+        if (settings.mktNavEnabled) {
+            var main = document.querySelector('main');
```
Comment (chuckharmston) [inline src/media/js/elements/nav.js]: Muse: we should attach `$('body > main')` to `z`.

File: src/media/js/elements/nav.js (hunk 1 above)
Comment (mstriemer) [inline src/media/js/elements/nav.js]: I think I'd rather not use jQuery if we don't need to. This is pretty straightforward and avoiding having to add `+ 'px'` doesn't seem like a big benefit.

**Comment by mstriemer** [comment]
Going to merge this. Happy to discuss our jQuery usage more but for now this is a regression that needs a fix.
